runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -7885,8 +7885,65 @@ done:;
 	VMINLINE VM_BytecodeAction
 	defaultvalue(REGISTER_ARGS_LIST)
 	{
+retry:
+		j9object_t newObject = NULL;
 		VM_BytecodeAction rc = EXECUTE_BYTECODE;
-		// TODO: Implement bytecode
+		U_16 index = *(U_16*)(_pc + 1);
+		J9ConstantPool *ramConstantPool = J9_CP_FROM_METHOD(_literals);
+		J9RAMClassRef *ramCPEntry = ((J9RAMClassRef*)ramConstantPool) + index;
```

Comment:
Can we use `const` for `index`, `ramConstantPool` and `ramCPEntry`? http://duramecho.com/ComputerInformation/WhyHowCppConst.html